### ISSUE TITLE
Duplicated columns in section property (xml)

### ISSUE THREAD

**Issue by ivansiiito** [ArtifexSoftware/pdf2docx#245]
Hi!

I have been using pdf2docx and I think it's probably the best solution available to convert .pdf documents into .docx. I truly like it.

Lately, I have been using pdf2docx in order to generate a .docx file, then apply several formatting changes and produce a new .pdf document. Doing so, I have realized that there might be a bug in the way the w:sectPr element for a section is created.

Let's say I have a .docx with 4 pages. Every page is converted into its own section, which is ok. However, if I check the fourth section in order to check its margins and so on, I get this XML:
```
<w:sectPr xmlns:w="http://schemas.openxmlformats.org/wordprocessingml/2006/main" xmlns:wpc="http://schemas.microsoft.com/office/word/2010/wordprocessingCanvas" xmlns:mo="http://schemas.microsoft.com/office/mac/office/2008/main" xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" xmlns:mv="urn:schemas-microsoft-com:mac:vml" xmlns:o="urn:schemas-microsoft-com:office:office" xmlns:r="http://schemas.openxmlformats.org/officeDocument/2006/relationships" xmlns:m="http://schemas.openxmlformats.org/officeDocument/2006/math" xmlns:v="urn:schemas-microsoft-com:vml" xmlns:wp14="http://schemas.microsoft.com/office/word/2010/wordprocessingDrawing" xmlns:wp="http://schemas.openxmlformats.org/drawingml/2006/wordprocessingDrawing" xmlns:w10="urn:schemas-microsoft-com:office:word" xmlns:w14="http://schemas.microsoft.com/office/word/2010/wordml" xmlns:wpg="http://schemas.microsoft.com/office/word/2010/wordprocessingGroup" xmlns:wpi="http://schemas.microsoft.com/office/word/2010/wordprocessingInk" xmlns:wne="http://schemas.microsoft.com/office/word/2006/wordml" xmlns:wps="http://schemas.microsoft.com/office/word/2010/wordprocessingShape" w:rsidR="00FC693F" w:rsidRPr="0006063C" w:rsidSect="00034616">
  <w:pgSz w:w="11906" w:h="16838"/>
  <w:pgMar w:top="704" w:right="1364" w:bottom="344" w:left="1412" w:header="720" w:footer="720" w:gutter="0"/>
  <w:cols w:space="720" w:num="1" w:equalWidth="0">
    <w:col w:w="9132" w:space="0"/>
    <w:col w:w="9132" w:space="0"/>
    <w:col w:w="9132" w:space="0"/>
    <w:col w:w="9132" w:space="0"/>
  </w:cols>
  <w:docGrid w:linePitch="360"/>
</w:sectPr>
```
As you can see, the w:cols element has an attribute w:num="1", while it includes 4 elements w:col inside. It seems that Microsoft Word gives more relevance to the value of w:num, and somehow ignores the three extra w:col elements. And so, the document seems fine.

However, if you use libreoffice to convert this .docx back to a .pdf, it seems libreoffice gives more importance to the number of w:col elements, and renders the document as having 4 columns, which is wrong.

Moreover, in a longer document, it happens that the section 5 has 5 columns, section 6 has 6 columns and so on, which is clearly suspicious.

Thanks!

**Comment by dothinking** [comment]
Thank you for reporting this difficult-to-detect issue.

The root cause is that I added a new column directly to `w:cols`, while docx seems to **inherit column settings** by default from the previous section. As you pointed out, this causes `w:col` to accumulate in each section.

The fix is to clear existing columns before adding a new one. I will implement this in the upcoming version, `v0.5.7`.

**Comment by dothinking** [comment]
Fixed in version `v0.5.7`.
